macaroonbakery/checkers.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+
+class Caveat:
+    '''Caveat represents a condition that must be true for a check to
```

Comment:
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings